tests/testthat/setup.R

```diff
@@ -1,3 +1,5 @@
+library(xml2) # This should be imported already
```

Comment:
Being outside the `test_that()` calls it needs to have `::` or imported with `library()`